pkg/apis/pingcap/v1alpha1/tidbmonitor_types.go

```diff
@@ -173,8 +173,17 @@ type TidbClusterRef struct {
 	Name string `json:"name"`
 }
 
-// TODO: sync status
+// TidbMonitorStatus is the current status of
 type TidbMonitorStatus struct {
+
+	// Ready is the number of ready containers
+	Ready corev1.ConditionStatus `json:"ready,omitempty"`
+
+	// Total number of non-terminated pods targeted by this deployment that have the desired template spec.
+	UpdatedReplicas int32 `json:"updatedReplicas,omitempty"`
+
+	// Total number of available pods (ready for at least minReadySeconds) targeted by this tidbMonitor.
+	AvailableReplicas int32 `json:"availableReplicas,omitempty"`
```

Comment:
maybe it's better to just add a  `DeploymentStatus` field to hold the underlying deployment status